src/App.ts

```diff
@@ -1075,50 +1142,6 @@ export default class App<AppCustomContext extends StringIndexed = StringIndexed>
       listenerArgs.respond = buildRespondFn(this.axios, body.response_urls[0].response_url);
     }
 
-    // Set ack() utility
-    if (type !== IncomingEventType.Event) {
-      listenerArgs.ack = ack;
-    } else {
-      // Events API requests are acknowledged right away, since there's no data expected
-      await ack();
```

Comment:
So where does this call (the auto-acknowledgement) happen now in this PR?

Reply:
The auto-acknowledgement behavior is moved to the [autoAcknowledge](https://github.com/slackapi/bolt-js/blob/9b194baa95f68c7361f99e49d1d755077c6c6b7f/src/middleware/builtin.ts#L125) middleware

This middleware is then added to the default list of listeners for the [event](https://github.com/slackapi/bolt-js/blob/9b194baa95f68c7361f99e49d1d755077c6c6b7f/src/App.ts#L610) and [message](https://github.com/slackapi/bolt-js/blob/9b194baa95f68c7361f99e49d1d755077c6c6b7f/src/App.ts#L679) (from the docs [view](https://tools.slack.dev/bolt-js/concepts/view-submissions) & [options](https://tools.slack.dev/bolt-js/concepts/options) do not need auto-acknowledgement behavior)

Reply:
Thanks for pointing that out! That was confusing for me 🥴